### ISSUE TITLE
Chrome crashing a lot

### ISSUE THREAD

**Issue by usamaiqbal1177** [ElderDrivers/EdXposed#809]
Chrome crashing a lot

**什么情况/What happened?**
Google Chrome browser is crashing a lot. I have disabled chrome in edxposed manager's whitelist as well but it still continues to crash. I have given all the permissions to chrome, Google play services and even play store. 

**Xposed模块列表/Xposed Module List**
DisabledFlagSecure, CorePatch, GravityBox R, Greenify, Scoop, SudoHide, XInternalSD, XposedSMSCode 

**Magisk模块列表/Magisk Module List**
Riru core, EdXposed 4672

**版本信息/Versions**

Android: 11

Magisk: 21401 canary

Riru: 23.3

EdXposed: master 4672

**相关日志/Related Logs**
https://del.dog/phybacowyp

 
[EdXposed_Verbose_20210121_133522.txt](https://github.com/ElderDrivers/EdXposed/files/5847884/EdXposed_Verbose_20210121_133522.txt)
[EdXposed_Modules_20210121_133503.txt](https://github.com/ElderDrivers/EdXposed/files/5847885/EdXposed_Modules_20210121_133503.txt)

**Comment by MlgmXyysd** [comment]
It seems to be a problem of permission.
Will this problem still occur after EdXposed is disabled?

**Comment by usamaiqbal1177** [comment]
> It seems to be a problem of permission.
> Will this problem still occur after EdXposed is disabled?

No. After Uninstalling EdXposed, it doesn't happen at all. However, now the crashes are less than they were before.
